src/pages/webhooks/commands.md

```diff
@@ -256,3 +278,55 @@ bin/magento webhooks:generate:module
 ```terminal
 Module was generated in the app/code/Magento directory
 ```
+
+## Emulate webhook execution
+
+The `webhooks:dev:run` command is used for development and testing purposes only. It emulates the execution of your registered webhook containing a custom payload with requiring changes to the Commerce application.
+Run this command after setting the initial webhook payload in a `webhooks.xml` file, Then run the command again any time you make subsequent modifications to the payload until you can confirm that the payload works as expected.
```

Comment:
This is very small and it may be disregarded if not necessary to change, but there's a capital T after a comma: "...payload in a webhooks.xml file, Then...". I think there should either be a period to keep the capital T there, or make it a lowercase t. I think second option makes more sense.

Reply:
@mariam21arauj Comments/PRs to fix typos are always welcomed. Thanks!